pkg/aaparser/aaparser.go

```diff
@@ -0,0 +1,45 @@
+package aaparser
+
+import (
+	"fmt"
+	"log"
+	"os/exec"
+	"strconv"
+	"strings"
+)
+
+// GetVersion returns the major and minor version of apparmor_parser
+func GetVersion() (int, int, error) {
+	// get the apparmor_version version
+	cmd := exec.Command("/sbin/apparmor_parser", "--version")
```

Comment:
I know the prior code inlined did the same thing, so this doesn't necessarily need to hold up the PR, but any reason we should `LookPath` the parser in case it ever gets installed somewhere else in the path?